### DISCUSSION TITLE
If we use both a sonic container and a scheduler container, do we need to specify `SEARCH_BACKEND_ENGINE` from both the main and the scheduler containers?

### DISCUSSION THREAD

**Discussion by melyux** [ArchiveBox/ArchiveBox#1189] [Q&A] [answered]
Or is just setting in the main container's ENV variables enough?

**Comment by pirate** [comment, marked as answer]
You need to specify it on both if you're using env vars, however an easier way is to set the config value persistently for all instances using `archivebox config --set KEY="VALUE"` (which is equivalent to adding a line in ArchiveBox.conf).
